### ISSUE TITLE
Logging null from custom scripting elides 'Script Log:' prefix 

### ISSUE THREAD

**Issue by shawnferry** [rangav/thunder-client-support#900]
**Describe the bug**

Logging null from custom scripting elides 'Script Log:' prefix 

**To Reproduce**

```
function  bugDemo() {
  thing = null
  console.log("foo")
  console.log(thing)
  console.log("bar")
}
```

```
Script Log: foo 
null
Script Log: bar 
```

**Expected behavior**

Logging null from a script to console.log should still be prefixed with 'Script Log:'

**Platform:** 
 - OS: OS X 13.0.1
 - vscode version: 1.73.0-insider
 - extension version: v2.0.2

**Solution:**

**Comment by rangav** [comment]
Thanks for reporting the issue, will look into it.

**Comment by rangav** [comment]
This Bug is fixed is now and published to marketplace, please update to `v2.1.0`

See all features released
https://github.com/rangav/thunder-client-support/releases/tag/v2.1.0

Please let me know your feedback.
